client/me/help/help-contact/index.jsx

```diff
@@ -39,7 +39,10 @@ import HappychatConnection from 'components/happychat/connection';
 import QueryOlark from 'components/data/query-olark';
 import QueryTicketSupportConfiguration from 'components/data/query-ticket-support-configuration';
 import HelpUnverifiedWarning from '../help-unverified-warning';
-import { sendChatMessage as sendHappychatMessage, sendUserInfo } from 'state/happychat/actions';
+import {
+	sendChatMessage as sendHappychatMessage,
+	sendUserInfo,
+} from 'state/happychat/connection/actions';
```

Comment:
Are these related to connection maybe should be moved in `chat/actions` ?

Reply:
Answered above: I think it makes sense to package them within `happychat/connection`.